go-controller/pkg/ovn/master.go

```diff
@@ -664,6 +664,7 @@ func (oc *Controller) ensureNodeLogicalNetwork(node *kapi.Node, hostSubnets []*n
 	var v4Gateway, v6Gateway net.IP
 	var hostNetworkPolicyIPs []net.IP
 	logicalRouterPortNetwork := []string{}
+	logicalSwitch.OtherConfig = map[string]string{}
```

Comment:
NIT: Is this really needed? I'd think that is the default value in `nbdb.LogicalSwitch` assigned in line 660.